src/screens/Lessons/screens/Lesson/components/LessonCard/components/WistiaVideo/index.js

```diff
@@ -1,25 +1,35 @@
-import React from 'react'
+import React, {Component} from 'react'
 import {noVideoDescriptionText} from 'utils/text'
 import {Paragraph} from 'egghead-ui'
 
-export default ({title, wistiaId}) => wistiaId
-  ? <div className='aspect-ratio aspect-ratio--16x9'>
-      <div className='aspect-ratio--object'>
-        <iframe
-          ref={(iframe) => {
-            if (iframe) {
-              iframe.contentWindow.location.replace(`//fast.wistia.net/embed/iframe/${wistiaId}?videoFoam=true`)
-            }
-          }}
-          width='100%'
-          height='100%'
-          frameBorder='0'
-          scrolling='no'
-          allowTransparency
-          allowFullScreen
-        />
-      </div>
-    </div>
-  : <Paragraph>
-      {noVideoDescriptionText}
-    </Paragraph>
+export default class extends Component {
+
+  shouldComponentUpdate(nextProps) {
+    return this.props.wistiaId !== nextProps.wistiaId 
+  }
```

Comment:
iframes behave oddly, so this ensures no flickering or issues when the wistiaId doesn't change